src/jestRunner.ts

```diff
@@ -58,6 +58,25 @@ export class JestRunner {
     await this.runTerminalCommand(command);
   }
 
+  public async runCurrentFileWithCoverage() {
```

Comment:
I would suggest to add an `options` parameter to the `runCurrentFile()` method, because there is some duplicated code now.

Reply:
I applied the feedback @viciteufel, thank you very much!